tools/sql_extraction/src/test/kotlin/DataFlowSolverTest.kt

```diff
@@ -39,13 +42,16 @@ class DataFlowSolverTest {
     }
 
     @Test
-    fun `solver throws without any valid frontends`() {
+    fun `solver returns empty sequence without any valid frontends`() {
```

Comment:
curious on the style too... why not 'SolverReturnsEmptySequence....'? Is this recommended?

Reply:
Since the language allows method names with spaces, I thought it would be more readable. I just did more reading and found [unit testing best practices for Kotlin](https://resources.jetbrains.com/storage/products/kotlinconf2018/slides/4_Best%20Practices%20for%20Unit%20Testing%20in%20Kotlin.pdf), which is recommending backticks.

Reply:
ok thanks!